<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Basma" failures="1" tests="3" name="testgroup(failed)" time="0.075" errors="0" timestamp="2024-10-13T17:52:01 BST">
  <testcase classname="Tests.TestingTestCases" name="test2" time="0.014"/>
  <testcase classname="Tests.TestingTestCases" name="test3" time="0.001"/>
  <testcase classname="Tests.TestingTestCases" name="test1" time="0.008">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Tests.TestingTestCases.test1(TestingTestCases.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="Tests.TestingTestCases" name="LoginTest" time="0.0">
    <ignored/>
  </testcase> <!-- LoginTest -->
</testsuite> <!-- testgroup(failed) -->
